### DISCUSSION TITLE
Small screen on Playwright - Changedetection

### DISCUSSION THREAD

**Discussion by SrPeterr** [community-scripts/ProxmoxVE#864] [Q&A]
Hello!

Well I'm having this problem using playwright that the visual filter selector screen looks very small... on some sites, on https://community-scripts.github.io/ProxmoxVE/ I see it like this:
<img width="1432" alt="brave_sG5cyvkhSB" src="https://github.com/user-attachments/assets/d259262a-0998-4337-a5c6-2cb60d30b4ec" />

But on https://github.com/community-scripts/ProxmoxVE:
<img width="1238" alt="brave_ZZRG51rOMq" src="https://github.com/user-attachments/assets/4af0a33f-9f7b-4715-b02f-22bc456da0a8" />

Playwright is set to: `ws://localhost:3000/chrome?launch={defaultViewport:{height:1920,width:1080},headless:false,stealth:true}&blockAds=true`

**Comment by SrPeterr** [comment]
Found this: https://github.com/dgtlmoon/changedetection.io/discussions/2739

I changed it and still the same

**Comment by MMinjin** [comment]
Change Detection (the Playwright portion at least) has been broken for a while. I had to roll back to v0.47.03 to get it to work. 

edit: there are some differences in the Playwrite driver environment between the versions...maybe related? In the install script under Creating Services.

**Reply by knario** [reply]
Hi. How did you manage to revert to a previous version?

**Reply by MMinjin** [reply]
I got lucky and had backups I could restore once I realized that it wasn't working correctly anymore. I now have a Change Detection container and a CD-test container where I am using the newest version to monitor when it gets fixed. Can't say I am savvy enough to know how to install an old version from scratch.

**Reply by knario** [reply]
I think I also keep an hdd with summer backups. let's see if I have any luck. :-)

**Comment by Koliera** [comment]
Anyone had luck with this? Kind of want to be able to select the appropriate part of the page to check.

**Comment by Koliera** [comment]
Looks like the script is using an older version env variables, there's a suggestion here - https://github.com/dgtlmoon/changedetection.io/discussions/2739
Anyone can see if an update can be pushed?
Specifically:
 environment: 
 #            - SCREEN_WIDTH=1920 
 #            - SCREEN_HEIGHT=1024 
 #            - SCREEN_DEPTH=16 
 #            - MAX_CONCURRENT_CHROME_PROCESSES=10

**Reply by 011V32** [reply]
Has the old/incorrect configuration been updated so that it works now?

**Comment by SrPeterr** [comment]
Any update on this? After installing completely new, still with a very small screen...
